firewood/tests/merkle.rs

```diff
@@ -16,7 +16,7 @@ fn merkle_build_test<K: AsRef<[u8]> + std::cmp::Ord + Clone, V: AsRef<[u8]> + Cl
     items: Vec<(K, V)>,
     meta_size: u64,
     compact_size: u64,
-) -> Result<MerkleSetup<Store>, DataStoreError> {
+) -> Result<MerkleSetup<Store, Bincode>, DataStoreError> {
```

Comment:
nit: I don't love the name here as it implies it has to be the bincode implementation, but I'm okay with this for now.